File: TRIANGLE PATTERN 1/code4.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter your no. of rows : ");
	scanf("%d",&row);

	for(int i = 1; i <= row ; i++){

		for(int j = 1 ; j <= i ;j++){

			if(j%2 == 1){

				printf("%d\t",j);
			}else{

				printf("A\t");
			}
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code4.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();

		for(int i = 0; i < row ; i++){
			for(int j = 1;j <=row ;j++){

				System.out.print((j+i)+"	");

			}
			System.out.println("\n");
		}
	}
}
